main.py

```diff
@@ -50,45 +50,30 @@ def main():
         # Set the input_file variable to the 2nd argument (input file path).
         input_file = sys.argv[1]
 
+        if not (os.path.isfile(input_file)):
+            print("ERROR: Path not valid.")
+
```

Comment:
Not a real change! I just moved this up out of personal preference

Reply:
No audio! Woooo I need to check. I think my test files have all been silent. Thanks for spotting this. -c:a is in the subprocess call so it should be there. I'll check this ASAP.
Many thanks @kfrn this is so much fun :)